phpunit/data/themedir1/block-theme-child-with-fluid-layout/theme.json

```diff
@@ -1,6 +1,6 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/theme.json",
-	"version": 2,
```

Comment:
I don't think there's a good way to avoid having to update the theme.json version in all of the fixtures.

Many of the tests use deep equivalence to check what they are testing which means the addition of `"defaultFontSizes": false` breaks those tests even though that's not what they're testing. Rather than going through and fixing all the tests, it seemed easier to just update the fixtures.